website/content/docs/connect/security.mdx

```diff
@@ -97,3 +97,25 @@ using a local Connect proxy. This is documented in the
 
 **If non-proxy traffic can communicate with the service**, this traffic
 will not be encrypted or authorized via Connect.
+
+### Restrict Access to Envoy's Administration Interface
+
+Envoy exposes a local, unauthenticated
+[administration interface](https://www.envoyproxy.io/docs/envoy/latest/operations/admin)
+that can be used to query and modify different aspects of the proxy. The admin
+interface allows retrieving potentially sensitive information such as:
+
+* Envoy configuration.
+* TLS certificates
+* List of upstream services and endpoints
+
+In addition, the admin interface allows performing operations which affect how
+connections are routed to or from the proxy within the service mesh.
```

Comment:
```suggestion
In addition, the admin interface allows you to perform operations that affect how
connections are routed to or from the proxy within the service mesh.
```
I understand that we avoid 'you' sometimes because the action might be performed programmatically or in some automated way, but I think we should failover to making 'you' the subject for clarity.